tools/ncf.py

```diff
@@ -135,7 +159,8 @@ def parse_bundlefile_metadata(content, bundle_type):
       res['class_parameter_id'] = res['bundle_args'].index(res['class_parameter'])+1
     except:
       res['class_parameter_id'] = 0
-      raise Exception("The class_parameter name \"" + res['class_parameter'] + "\" does not seem to match any of the bundle's parameters")
+      name = res['bundle_name'] if bundle_name in res else "unknown"
```

Comment:
That will not work, it should be 

     if 'bundle_name' in res else "unknown"